# Modify the Makefile so that it does the following
# - Disables UI generation (we use the pre-generated UI, otherwise this fails on s390x, ppcl64e, & x86)
# - Disables race condition testing, as it currently fails breaking the build.
--- a/Makefile	2025-01-18 15:46:05.000000000 -0500
+++ b/Makefile	2025-01-19 11:31:57.610839843 -0500
@@ -193,7 +193,7 @@
 build-ui: ## Build UI
 	(cd web/; pnpm install --frozen-lockfile; pnpm build)
 
-build-server: build-ui generate-openapi ## Build server
+build-server: ## Build server
 	CGO_ENABLED=${CGO_ENABLED} GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -tags '$(TAGS)' -ldflags '${LDFLAGS}' -o ${DIST_DIR}/woodpecker-server${BIN_SUFFIX} go.woodpecker-ci.org/woodpecker/v3/cmd/server
 
 build-agent: ## Build agent
